src/main/webapp/app/programming-assessment/manual-result/programming-assessment-manual-result.service.ts

```diff
@@ -32,6 +34,24 @@ export class ProgrammingAssessmentManualResultService {
             .map((res: EntityResponseType) => this.resultService.convertDateFromServer(res));
     }
 
+    /**
+     * Send request to update the assessment after the complaint
```

Comment:
This is still not a good documentation. Please just describe very briefly what "updating" entails. What happens to the assessment? What exactly gets updated? Does it get updated all the time, or are there certain conditions? 
Also the parameters: I know that a feedback array is a list of feedback items, but how does this relate to the other parameters or the whole method? 

We are all guilty of often writing redundant documentation, but it would be great if we started trying to put ourselves in the shoes of a new developer in the project.

Reply:
This is especially for the service methods since they are shared between multiple components and will be often reused in the future.

Reply:
You are right about documenting the shared services. I updated the documentation, so it should be easier to understand now. In my opinion, though I don't see a big problem here since if you see the service and what URL it is using, you will navigate to check the server code to see what is happening on the REST controller.